Misc/NEWS.d/next/Library/2020-02-06-13-34-52.bpo-39350.wRwup1.rst

```diff
@@ -0,0 +1,2 @@
+Fix regression in :class:`fractions.Fraction` if the numerator and/or the
```

Comment:
I think there's more to say here.

This _does_ fix a regression in 3.9.0a3, but it also represents a behaviour change from 3.8.x to 3.9.x. We should document that behaviour change, at least in the news entry.